OSVR-Unity/Assets/OSVRUnity/src/VRHead.cs

```diff
@@ -63,12 +62,12 @@ void Start()
             {
                 Init();
                 CatalogEyes();
-                _distortionEffect = GetComponent<OsvrDistortion>();
+/*
```

Comment:
Mono may be important if you just want to see a preview on a desktop screen

Or if you have an HMD that accepts one input and shows it on both eyes.

Yuval


On 8/17/2015 11:00 AM, Ryan Pavlik wrote:
>
> In OSVR-Unity/Assets/OSVRUnity/src/VRHead.cs 
> <https://github.com/OSVR/OSVR-Unity/pull/70#discussion_r37196538>:
>
> > @@ -63,12 +62,12 @@ void Start()
> >              {
> >                  Init();
> >                  CatalogEyes();
> > -                _distortionEffect = GetComponent<OsvrDistortion>();
> > +/*
>
> I didn't know if the |viewMode == mono| thing was important, given 
> Unity's somewhat unique case of having the editor, the play mode in 
> editor, and standalone mode. If it were just me, I'd remove it, but I 
> didn't want to step on toes.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OSVR/OSVR-Unity/pull/70/files#r37196538>.
>